tools/excludedFolders.js

```diff
@@ -1 +1,4 @@
-module.exports = process.env.IS_V2 === "true" ? ["helpers", ".DS_Store"] : ["helpers", ".DS_Store", "v2"];
+module.exports =
+  process.env.IS_V2 === "true"
+    ? ["helpers", "hooks", ".DS_Store"]
+    : ["helpers", ".DS_Store", "v2", "hooks"];
```

Comment:
Not blocking, just a suggestion to be more easier to read:
```suggestion
  process.env.IS_V2 === "true"
    ? [".DS_Store", "helpers", "hooks"]
    : [".DS_Store", "helpers", "hooks", "v2"];
```